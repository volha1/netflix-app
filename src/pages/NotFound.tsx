import React, { ReactElement } from 'react';

const NotFound = (): ReactElement => {
  return (
    <div className="not-found">
      <h1>Sorry, page was not found</h1>
    </div>
  );
};

export default NotFound;
