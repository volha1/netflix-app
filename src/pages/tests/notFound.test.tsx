import React from 'react';
import renderer from 'react-test-renderer';

import NotFound from '../NotFound';

describe('NotFound test', () => {
  it('NotFound snapshot', () => {
    const component = renderer.create(<NotFound />);
    const tree = component.toJSON();

    expect(tree).toMatchSnapshot();
  });
});
