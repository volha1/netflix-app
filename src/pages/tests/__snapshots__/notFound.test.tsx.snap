// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound test NotFound snapshot 1`] = `
<div
  className="not-found"
>
  <h1>
    Sorry, page was not found
  </h1>
</div>
`;
